applications/luci-app-cifs-mount/root/etc/init.d/cifs

```diff
@@ -42,8 +42,8 @@ mount_natshare() {
 	config_get smbver $1 smbver
 	
 	mkdir -p $natpath && chmod 777 $natpath
-	#echo "mount -t cifs -o vers=$smbver,user=$users,password=$pwd,iocharset=$iocharset,$agm //$server/$name $natpath"
-	mount -t cifs -o vers=$smbver,user=$users,password=$pwd,iocharset=$iocharset,$agm //$server/$name $natpath
+	#echo "mount -t cifs -o vers=$smbver,user=$users,username=$users,pass=$pwd,password=$pwd,iocharset=$iocharset,$agm //$server/$name $natpath"
+	mount -t cifs -o vers=$smbver,user=$users,username=$users,pass=$pwd,password=$pwd,iocharset=$iocharset,$agm //$server/$name $natpath
```

Comment:
Anyway `pass` should be useless right? Please remove it.

Reply:
done